source/Gesture Recognizer/Assets/Plugins/GestureRecognizer/SerializablePoint.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+using PDollar = PDollarGestureRecognizer;
+
+namespace GestureRecognizer
+{
+    [Serializable]
+    public class SerializablePoint : PDollar.Point
+    {
+        #region CONSTRUCTORS
+
+        public SerializablePoint(float x, float y, int strokeId) : base(x, y, strokeId) { }
```

Comment:
Empty constructors should have the brackets in different line but both in the same line.

Reply:
Done